src/osm_osw_reformatter/osm2osw/osm2osw.py

```diff
@@ -11,16 +11,19 @@ def __init__(self, prefix: str, pbf_file=None, workdir=None):
         self.pbf_path = str(Path(pbf_file))
         filename = os.path.basename(pbf_file).replace('.pbf', '').replace('.osm', '')
         self.workdir = workdir
-        self.filename = f'{prefix}.{filename}'
+        self.filename = f'{prefix + "." if prefix else ""}{filename}'
```

Comment:
is this causing the issue on the output? Earlier version used to output zip file.

Reply:
This change simply removes the period from the beginning of the file name if no prefix is passed.  On the local branch I ran:

git checkout 984daf7 (the revision at the beginning of this feature)

and I still get only 3 geojson files